test/tests/test_sql_fields.py

```diff
@@ -142,7 +142,12 @@ def test_iexact_lookup(self):
         )
 
     def test_search_lookup_fails(self):
-        with self.assertRaises(NotImplementedError):
+        if VERSION >= (2, 0):
+            expected = FieldError
+        else:
+            expected = NotImplementedError
+
+        with self.assertRaises(expected):
```

Comment:
To be honest, I don't really understand why we're expecting `NotImplementedError` to be thrown in 1.x series, but there _seems_ to be a [reason behind it](https://github.com/jimfunk/django-postgresql-netfields/blob/4c57f0434fd43ee717f23697d141e945dd91ec62/netfields/lookups.py#L16-L23) that relates to Django 1.9.  But then again, the exception that is being raised here *isn't even* the same `NotImplementedError` that is raised on the linked line. So I'm confused.

So maybe this test is broken? I don't know. :-)

If anyone has a suggestion on how to fix it or better test it, I'm totally up for implementing it.

Reply:
Django 1.9 raised NotImplementedError here, and that was set up to emulate 1.9 behaviour in earlier versions. If it raises FieldError now, the InvalidLookup could be used instead and InvalidSearchLookup removed.